index.md

```diff
@@ -4,7 +4,7 @@ title: "FOSDEM &lt;3 Ruby"
 subtitle: "The #1 Ruby language devroom at FOSDEM"
 ---
 
-### [Call for proposals now open!](/cfp)
+### [Call for proposals now open!](/cfp.html)
```

Comment:
somehow without it wasn't working on my local machine so I changed all of them to have the .html